src/Spain/Mews.Fiscalizations.Spain/Client.cs

```diff
@@ -16,7 +16,7 @@ public class Client
         public Client(X509Certificate certificate, Environment environment, TimeSpan httpTimeout)
         {
             var domain = environment.Match(
-                Environment.Test, _ => "www7.aeat.es",
+                Environment.Test, _ => "prewww1.aeat.es",
```

Comment:
Source: https://preportal.aeat.es/PRE-Exteriores/Inicio/Inicio.html

https://www7.aeat.es se migra a https://prewww1.aeat.es equivalente en cuanto a requisitos a https://www1.agenciatributaria.gob.es

which means `https://www7.aeat.es` is now replaced with `prewww1.aeat.es`